libzecale/circuits/aggregator_circuit.hpp

```diff
@@ -20,23 +20,16 @@ namespace libzecale
 /// nested proofs. Inputs are allocated as follows:
 ///
 ///   <hash of nested verification key>
-///   <input[1,1]>
+///   <packed_results>
+///   <nested_inputs[1]>
+///   <nested_inputs[2]>
 ///   ...
-///   <input[1,M]>
-///   <result[1]>
-///   ...
-///   ...
-///   ...
-///   <input[N,1]>
-///   ...
-///   <input[N,M]>
-///   <result[N]>
+///   <nested_inputs[N]>
 ///
 /// where:
 ///   N = NumProofs,
-///   M = num_inputs_per_nested_proof,
-///   input[i,j] = j-th input to i-th proof,
-///   result[i] = result of i-th proof verification)
+///   nested_inputs[i] = j-th input to i-th proof,
+///   packed_results = result of i-th proof verification as bits)
```

Comment:
The ending parenthesis seems extraneous.
`packed_results = result of i-th proof verification as bits` -> `packed_results = field element representing the packed "result proof verification bits" of all the proofs in the batch`

Reply:
updated comments.  thanks.